docs/developer/meta/config-specs.md

```diff
@@ -47,7 +47,7 @@ Every file has 3 possible attributes:
 Every option has 10 possible attributes:
 
 - `name` - This is the name of the option (**REQUIRED**)
-- `description` - Information about the option (**REQUIRED**)
+- `description` - Information about the option. This can be a multi-line string but each line must be less than 120 characters (**REQUIRED**)
```

Comment:
```suggestion
- `description` - Information about the option. This can be a multi-line string, but each line must contain fewer than 120 characters (**REQUIRED**).
```